onadata/apps/api/tests/management/commands/test_delete_users.py

```diff
@@ -56,3 +57,30 @@ def test_delete_users_no_input(self, mock_input):  # pylint: disable=R0201
             "User account 'barbie' has 0 projects, "
             "0 forms and 0 submissions. "
             "Do you wish to continue deleting this account?")
+
+    def test_user_details_are_available(self):
+        """
+        Test that a user's username and email are
+        no longer associated with a user account after
+        the account is soft-deleted
+        """
+        user = User.objects.create(
+            username="Ken",
+            email="ken@gmail.com")
+        username = user.username
+        email = user.email
+        out = StringIO()
+        sys.stdout = out
+        new_user_details = [username+':'+email]
+        call_command(
+            'delete_users',
+            user_details=new_user_details,
+            user_input='True',
+            stdout=out)
+
+        self.assertEqual(
+            "User Ken deleted successfully.",
+            out.getvalue())
+
+        with self.assertRaises(ObjectDoesNotExist):
```

Comment:
Tiny thing/change.... Please use `User.DoesNotExist`

Reply:
Cool cool. Changed that.